server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+// const mongoose = require('mongoose');
+const app = express();
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+
+
+const orderController = require('./controllers/orderController');
+require('./database/db')
+const PORT = 3000;
+
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(methodOverride('_method'));
+app.use('/orders', orderController);
+
```

Comment:
it will work fine the way you have it, but i suggest reordering this slightly to group similar things together:

```
// db
require('./database/db')

// middleware
app.use(bodyParser.urlencoded({extended: false}));
app.use(methodOverride('_method'));

// controllers
const orderController = require('./controllers/orderController');
app.use('/orders', orderController);
```